docs/iceberg-rest-service.md

```diff
@@ -388,13 +388,28 @@ SELECT * FROM dml.test;
 You could run Gravitino Iceberg REST server though docker container:
 
 ```shell
-docker run -d -p 9001:9001 apache/gravitino-iceberg-rest:0.6.0
+docker run -d -p 9001:9001 apache/gravitino-iceberg-rest:0.7.0-incubating
 ```
 
-Or build it manually to add custom logics:
+Gravitino Iceberg REST server in docker image could access local storage, you could change the configuration by environment variables to access S3 or GCS storage, please refer to storage section for more details.
```

Comment:
Gravitino Iceberg REST server in docker image could access local storage by default, however,  you should add the following variables the storage is cloud/remote storage like S3, xxxx,xxxxx

Reply:
updated